suite-native/theme/src/InvertedThemeProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from 'react';
+import { ThemeProvider } from 'react-fela';
+
+import { prepareNativeTheme } from '@trezor/theme';
+
+import { useGetSystemColorScheme } from './useGetSystemColorScheme';
+import { useUserColorScheme } from './useUserColorScheme';
+
+type InvertedThemeProviderProps = {
+    children: ReactNode;
+};
+
+export const InvertedThemeProvider = ({ children }: InvertedThemeProviderProps) => {
+    const { userColorScheme } = useUserColorScheme();
+    const systemColorScheme = useGetSystemColorScheme();
+    const activeColorScheme = userColorScheme === 'system' ? systemColorScheme : userColorScheme;
```

Comment:
Why not have all of this within one hook?

Reply:
Good idea ec7bdb40a